relay-filter/src/generic.rs

```diff
@@ -0,0 +1,148 @@
+//! Implements generic filtering based on the [`RuleCondition`] DSL.
+//!
+//! Multiple generic filters can be defined and they are going to be checked in FIFO order. The
+//! first one that matches, will result in the event being discarded with a [`FilterStatKey`]
+//! identifying the matching filter.
+
+use crate::{FilterStatKey, GenericFiltersConfig};
+use relay_event_schema::protocol::Event;
+use relay_protocol::RuleCondition;
+
+/// Maximum supported version of the generic filters schema. If the version in the project config
+/// is higher, no generic filters are applied.
```

Comment:
```suggestion
/// Maximum supported version of the generic filters schema.
///
/// If the version in the project config is higher, no generic filters are applied.
```
From [internal guidelines](https://www.notion.so/sentry/Relay-Code-and-Architecture-4fac481cf27741aba54d39594f1cae58?pvs=4#987c0416a9f54674999eed59eecbf846).